framework/src/org/apache/cordova/CordovaInterface.java

```diff
@@ -69,4 +69,7 @@ Licensed to the Apache Software Foundation (ASF) under one
      * Returns a shared thread pool that can be used for background tasks.
      */
     public ExecutorService getThreadPool();
+
+    public void requestPermission(CordovaPlugin plugin, int requestCode, String permission);
```

Comment:
JavaDoc would be great here - since this is public API. Eventually we should figure out how to publish the public API docs online.